lizard_connector/__init__.py

```diff
@@ -1 +1,4 @@
 # package
+import lizard_connector.jsdatetime
```

Comment:
Waarom importeer je zaken in de `__init__.py`? Normaliter laten we die bij N&S altijd leeg.

En *als* er iets instaat is dat vaak een reden om het over het hoofd te zien bij het debuggen...